src/filters/util.js

```diff
@@ -3,26 +3,18 @@
 angular.module('openshiftCommonUI')
   .filter("toArray", function() {
     return function (items) {
-      if (!items) {
-        return [];
-      }
-
-      if (angular.isArray(items)) {
-        return items;
-      }
-
-      var itemsArray = [];
-      angular.forEach(items, function (item) {
-        itemsArray.push(item);
-      });
-
-      return itemsArray;
+      return _.toArray(items);
+    };
+  })
+  .filter('size', function() {
+    return function(items) {
+      return _.size(items);
     };
   })
-  .filter('hashSize', function() {
-    return function(hash) {
-      if(!hash) { return 0; }
-      return Object.keys(hash).length;
+  .filter('hashSize', function($log) {
+    return function(items) {
+      $log.warn('hashSize filter is now deprecated. Use size filter instead.');
+      return _.size(items);
```

Comment:
Also a little nervous about changing the implementation without more testing since this is so heavily used. If they're subtly different, it could introduce bugs in working code. I suggest we leave it as-is and try to update the code to use the new filter over time.